docs/auth.md

```diff
@@ -1,178 +1,53 @@
-2. **Enable RBAC for Your API**:
-   - Go to **Dashboard > Applications > APIs > Your API**
-   - In the **Settings** tab, enable:
-     - **"Add Permissions in the Access Token"**
-     - **"RBAC"** (Role-Based Access Control)
-     - **"Add Roles in the Access Token"**
-
-> [!NOTE]
->
-> For the Auth0 Management API that is added by default, it is not necessary to enable RBAC; the previous step can be skipped.
-
-3. **Authorize Management API Access**:
-   - Go to **Dashboard > Applications > APIs > Auth0 Management API**
-   - Navigate to **"Machine to Machine Applications"** tab
-   - Find your application in the list
-   - **Authorize** your application
-   - Select the required scopes:
-     - `read:users` - to fetch user information
-     - `read:user_idp_tokens` - to read user roles
-
-## Role Management
-
-### Creating Roles
-
-1. Navigate to **User Management > Roles** in the Auth0 dashboard
-2. Click **"+ Create Role"** (blue button in the top right)
-3. Enter role details:
-   - **Name**: Admin, Member, or Viewer
-   - **Description**: Brief description of permissions
-
-![Auth0 Roles Management](roles.png)
-
-*Screenshot showing the existing roles (Admin, Member, Viewer) and the "+ Create Role" button in the Auth0 dashboard*
-
-**Existing Roles in the System:**
-- **Admin**: "can access anything a member can, plus /config"
-- **Member**: "can access both unrestricted and restricted views routes"  
-
-### Assigning Roles to Users
-
-1. Go to **User Management > Users**
-2. Click on the user's name
-3. Navigate to the **"Roles"** tab
-4. Click **"Assign Role"**
-5. Select the appropriate role(s)
-
-**Note**: Users can have multiple roles, but it's recommended to assign only the highest-level role needed (e.g., assign Admin and remove Member role).
-
-### Viewing User Roles
-
-You can view assigned roles using the Auth0 dashboard or Management API:
-
-**Dashboard Method**:
-1. User Management → Users → [User Name] → Roles tab
-
-**Management API Method**:
-```bash
-curl --request GET \
-  --url 'https://{yourDomain}/api/v2/users/USER_ID/roles' \
-  --header 'authorization: Bearer MGMT_API_ACCESS_TOKEN'
-```
-
-## API References
-
-### Auth0 Documentation References
-
-- [Configure Core RBAC](https://auth0.com/docs/manage-users/access-control/configure-core-rbac) - Core RBAC setup guide
-- [Register APIs](https://auth0.com/docs/get-started/auth0-overview/set-up-apis) - How to register and configure APIs
-- [View User Roles](https://auth0.com/docs/manage-users/access-control/configure-core-rbac/rbac-users/view-user-roles) - Managing user roles
-- [Get User Roles Endpoint](https://auth0.com/docs/api/management/v2#!/Users/get_user_roles) - Management API reference
-
-### Management API Endpoints Used
-
-1. **Get User by Email**:
-   ```
-   GET /api/v2/users-by-email?email={email}
-   ```
-
-2. **Get User Roles**:
-   ```
-   GET /api/v2/users/{user_id}/roles
-   ```
-
-3. **Get Management API Token**:
-   ```
-   POST /oauth/token
-   ```
```

Comment:
Sorry, maybe I was unclear. But we are now deleting a bunch of stuff that was already added and I think valuable.

My earlier comment about cleaning up these docs was in regards to new implementation details that were added.